docs/environment-docs.yml

```diff
@@ -25,3 +25,4 @@ dependencies:
       # See: https://github.com/geojupyter/jupytergis/issues/585
       - ../python/jupytergis_core
       - ../python/jupytergis_lab
+      - ../python/jupytergis
```

Comment:
```suggestion
```
Suggested by @martinRenou to fix the lite test.

Reply:
I added this because I updated the docs to generate API docs from `jupytergis` instead of `jupytergis_lab`. The end-user may be confused about why they're importing `jupytergis_lab` instead of `jupytergis`. I will change it back for now and open an issue to come back to this later.

Reply:
#661

Reply:
0b2eab4